package.json

```diff
@@ -33,7 +33,7 @@
     "test": "NODE_ENV=test BASE_URL=/console jest",
     "test:clear": "jest --clearCache",
     "test-coverage": "NODE_OPTIONS='--max-old-space-size=8192' NODE_ENV=test BASE_URL=/console jest --coverage",
-    "test-release": "npx semantic-release --debug --no-ci --test-run"
+    "test-release": "yarn semantic-release --debug --no-ci --test-run"
```

Comment:
since the github action uses `npx semantic-release` (see `checks.yml`) let's keep it the same